.github/workflows/actions/diff/action.yml

```diff
@@ -0,0 +1,95 @@
+name: Build and diff
+
+outputs:
+  changes:
+    description: 'Returns true if build output has changes'
+    value: ${{ steps.diff.outputs.changes || 'true' }}
+
+runs:
+  using: composite
+
+  steps:
+    - name: Set up diff drivers
+      shell: bash
+      run: |
+        npm install -g js-beautify
+        git config diff.minjs.textconv "js-beautify --editorconfig --type js"
+        git config diff.mincss.textconv "js-beautify --editorconfig --type css"
```

Comment:
Not clear to me what the function of the diff config (and thus `js-beautify`) here is.  All we see at the end of the process is an exit code, the diff format shouldn't matter.

Reply:
Yeah, it's worth avoiding having to wait for it in my mind.